opm/simulators/linalg/setupPropertyTree.cpp

```diff
@@ -115,7 +115,7 @@ setupPropertyTree(FlowLinearSolverParameters p, // Note: copying the parameters
     // No valid configuration option found.
     OPM_THROW(std::invalid_argument,
               conf + " is not a valid setting for --linear-solver-configuration."
-              " Please use ilu0, dilu, cpr, cpr_trueimpes, cpr_quasiimpes or isai");
+              " Please use ilu0, dilu, cpr, cprw, pr_trueimpes, cpr_quasiimpes cpr_trueimpesanalytic or isai");
```

Comment:
I guess there are a couple of misprints here.  I believe there's a `c` missing in `pr_trueimpes`, and that there should be a comma between `cpr_quasiimpes` and `cpr_trueimpesanalytic`.

Reply:
Ouch, will fix!

Reply:
Amended.